test_unstructured_ingest/test-ingest-confluence2.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/.. || exit 1
+
+# Tests the scenario where --confluence-list-of-spaces has been provided, with
+# a big number of documents in the spaces.
+OUTPUT_FOLDER_NAME=confluence2
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+
+if [ -z "$CONFLUENCE_USER_EMAIL" ] || [ -z "$CONFLUENCE_API_TOKEN" ]; then
+   echo "Skipping Confluence ingest test because the CONFLUENCE_USER_EMAIL or CONFLUENCE_API_TOKEN env var is not set."
+   exit 0
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --download-dir "$DOWNLOAD_DIR" \
+    --confluence-url https://unstructured-ingest-test.atlassian.net \
+    --confluence-user-email "$CONFLUENCE_USER_EMAIL" \
+    --confluence-api-token "$CONFLUENCE_API_TOKEN" \
+    --confluence-num-of-spaces 10 \
+    --confluence-list-of-spaces testteamsp1 \
```

Comment:
I thought per your text in main.py that one is not supposed to include both num-of-spaces and and list-of-spaces args?

Reply:
Yes, this part in the test checks if the connector can act the right way even when it is being used in the wrong way 
(checks if it can ignore the redundant arg and go with the preferred one)

If we'd like to keep each test checking exactly one aspect rather than testing multiple aspects with one test, I can remove the redundant arg there

I also added an explanation to avoid confusion if we keep the part:

https://github.com/Unstructured-IO/unstructured/blob/541497408523c799544b4ccdad87be61b35478a5/test_unstructured_ingest/test-ingest-confluence-large.sh#L20

Reply:
I think with explanation this is gtg